src/services/cheWorkspaceClient/index.ts

```diff
@@ -115,18 +115,14 @@ export class CheWorkspaceClient {
     this.baseUrl = baseUrl;
   }
 
-  setWebsocketContext(websocketContext: string): void {
-    this.websocketContext = websocketContext;
-  }
-
   updateRestApiClient(): void {
     const baseUrl = this.baseUrl;
     const headers = this.token ? { Authorization: `Bearer ${this.token}` } : {};
     this._restApiClient = WorkspaceClient.getRestApi({ baseUrl, headers });
   }
 
   async updateJsonRpcMasterApi(): Promise<void> {
-    let jsonRpcApiLocation = this.originLocation.replace('http', 'ws') + this.websocketContext;
+    let jsonRpcApiLocation = this.originLocation.replace('http', 'ws') + '/api/websocket';
```

Comment:
`websocketContext` should not be defined on the Dashboard side, it more should be on WorkspaceClient level. @akurinnoy and I think that it's OK to hard-code this value on WorkspaceClient side, in the same way as there is like /api/workspace and all related methods paths.

But due to backward compatibility reasons, it's done quickly in this way. Later it can be really fully moved to Che Workspace Client.